cmd/mothership/local/options.go

```diff
@@ -112,33 +113,66 @@ func componentsFromStrings(list []string, values []string) ([]*keb.Component, er
 	return comps, nil
 }
 
+func componentsFromClusterState(cluster cluster.State, values []string) ([]*keb.Component, error) {
```

Comment:
It would be nice to have a unit test for this function

Reply:
good idea! thanks